Makefile

```diff
@@ -138,3 +137,7 @@ test: PATH := $(WORKING_DIR)/bin:$(PATH)
 test:
 	@export PATH
 	cd examples && go test -v -json -tags=all -timeout 2h . 2>&1 | tee /tmp/gotest.log | gotestfmt
+
+renovate:	build
```

Comment:
How very optimistic of you :)

Reply:
Yes will probably have to downgrade to some form of `generate`.